pkg/controller/statusmanager/status_manager.go

```diff
@@ -52,6 +52,7 @@ const (
 	CertificateSigner
 	InfrastructureConfig
 	DashboardConfig
+	MachineConfig
```

Comment:
I think this list is ordered from from more important to least important problems. So I would say this probably needs to be above PodDeployment.